controlplane/pkg/plugins/utils.go

```diff
@@ -38,6 +39,32 @@ func StartPlugin(name, registry string, services map[plugins.PluginCapability]in
 		return err
 	}
 
+	go func() {
+		registered := true
+		for ; ; time.Sleep(5 * time.Second) {
+			resp, err := http.Get("http://localhost:5555/liveness")
+			if err != nil {
+				logrus.Errorf("failed to get NSMD liveness: %v", err)
+			}
+
+			nsmdAlive := err == nil && resp.StatusCode >= 200 && resp.StatusCode < 400
```

Comment:
We consider success only in case of `StatusCode` equals to 200

Reply:
- [ ] Only allow 200 return code for liveness probe